.gitignore

```diff
@@ -20,6 +20,9 @@ logs/modperl_error_log
 
 # Local databases
 data/mongodb
+Lang.openfoodfacts.*
+users_emails.sto
+html/data/top_translators.csv
```

Comment:
Could we ignore html/data/ as a whole? There's other scripts that export MongoDB, RDF, etc. there.

Reply:
Ok, done.